vm_dump.c

```diff
@@ -1107,6 +1107,19 @@ rb_vm_bugreport(const void *ctx, FILE *errout)
         }
     }
 
+    // Print a "minimal" crash report when errout is a TTY
+    int full_crash_report = !(isatty(fileno(errout)));
+    // RUBY_FULL_CRASH_REPORT takes precedence over the default
+    const char *ruby_full_crash_report_env = getenv("RUBY_FULL_CRASH_REPORT");
+    if (ruby_full_crash_report_env != NULL) {
+        if (strcmp(ruby_full_crash_report_env, "0") == 0) {
+            full_crash_report = false;
+        }
+        else {
+            full_crash_report = true;
+        }
```

Comment:
This makes `RUBY_FULL_CRASH_REPORT= ruby crash.rb` full dump.
I'd expect that empty value would disable the feature.
```suggestion
        full_crash_report = strtoul(ruby_full_crash_report_env, 0, 0) != 0;
```

Or even:
```suggestion
    bool full_crash_report = ruby_full_crash_report_env  ?
        strtoul(ruby_full_crash_report_env, 0, 0) != 0 :
        !isatty(fileno(errout));
```

Reply:
Thank you! I agree that `RUBY_FULL_CRASH_REPORT= ruby crash.rb` should disable the feature.
Fixed in 4a3ebd4a4af7574a88bce7db2d344c2edfd0cc68.